<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaiaResizer</class>
 <widget class="QMainWindow" name="MaiaResizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaiaResizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="line_src">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>5</y>
      <width>405</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Image source path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>5</y>
      <width>77</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Source dir:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="b_src">
    <property name="geometry">
     <rect>
      <x>492</x>
      <y>5</y>
      <width>27</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Browse image source directory</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>72</y>
      <width>333</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Operation status</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>71</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_counter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>72</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>-- / --</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="boxOperations">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>38</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>RGB --&gt; Gray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resize</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Invert</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>38</y>
      <width>67</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Settings:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxFilters">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>38</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>*.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*.jpg</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineWidth">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>38</y>
      <width>78</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineHeight">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>38</y>
      <width>77</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
